docs/concept-workflows.md

```diff
@@ -150,25 +150,25 @@ There is no default value and one must be supplied if a Retry Policy is provided
 #### Backoff coefficient
 
 - **Description**: Retries can occur exponentially.
-The backoff coefficient specifies how fast the retry interval is growing.
-The default is backoff coefficient is 2.0.
+The backoff coefficient specifies how fast the retry interval will grow.
+The default value is set to 2.0.
 A backoff coefficient of 1.0 means that the retry interval will always equal the [initial interval](#initial-interval).
 - **Use-case**: Use this to grow the interval between retries.
-By having a backoff coefficient, the first few retries happen relatively quickly to overcome intermittent failures, but subsequent retries will happen farther and farther apart to account for longer lasting outages.
+By having a backoff coefficient, the first few retries happens relatively quickly to overcome intermittent failures, but subsequent retries will happen farther and farther apart to account for longer lasting outages.
 Use the [maximum interval option](#maximum-interval) to prevent the coefficient from growing the retry interval too much.
 
 #### Maximum interval
 
 - **Description**: Specifies the maximum interval between retries.
-The default is 100x that of [initial interval](#initial-interval)
+The default is 100x that of [initial interval](#initial-interval).
 - **Use-case**: This is useful for coefficients greater than 1.0 as it prevents the interval from growing exponentially infinitely.
 
 #### Maximum attempts
 
 - **Description**: Specifies the maximum number of attempts that can be made to execute a Workflow in the presence of failures. If this limit is exceeded, the Workflow fails without retrying again.
-The default is unlimited and setting it to 0 also means unlimited.
-- **Use-case**: The can be used to ensure that retries do not continue indefinitely.
-However, in the majority of cases, we recommend relying on the [execution timeout](#execution-timeout) to limit the duration of the retries instead of the maximum attempts.
+The default is unlimited. Setting it to 0 also means unlimited.
+- **Use-case**: This can be used to ensure that retries do not continue indefinitely.
+However, in the majority of cases, we recommend relying on the [execution timeout](#execution-timeout) to limit the duration of the retries instead this.
```

Comment:
```suggestion
However, in the majority of cases, we recommend relying on the [execution timeout](#execution-timeout) to limit the duration of the retries instead of this.
```

Reply:
Yikes, can't believe I missed this. :/